client/utils/formatter.js

```diff
@@ -0,0 +1,11 @@
+export const capitalizeFirstLetterOfWords = (
+    sentence,
+    splitChar = ' ',
+    joinChar = ' '
+) => {
```

Comment:
This one is totally optional, but what do you think about using a "options" object - that way, when you use it, it would be like:

```js
capitalizeFirstLetterOfWords(sentence, { joinChar: '_' })
```

Which allows readers to understand which options are passed, and makes it harder to mess up the order of arguments.

Reply:
That approach is fine with me